### DISCUSSION TITLE
Convert float to int

### DISCUSSION THREAD

**Discussion by xandaaah** [NicoNex/tau#51] [Q&A] [answered]
How do I convert a float to an int? (assuming it is an integer in math terms)

**Comment by NicoNex** [comment, marked as answer]
Hi, sorry for the late answers but I'm out of town this weekend, you can just do `int(myfloat)`.

**Reply by xandaaah** [reply]
> Hi, sorry for the late answers but I'm out of town this weekend, you can just do `int(myfloat)`.

Thanks!
